portal/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -31,3 +41,6 @@ class ProjectService(projectRepository: ProjectRepository)(implicit executionCon
     } yield getProject
 
 }
+
+case class ProjectNotFoundException(private val message: String = "Project not found") extends RuntimeException(message)
+case class ProjectAccessDeniedException(private val message: String = "Access denied") extends RuntimeException(message)
```

Comment:
Making a `case class` with private fields in the constructor doesn't make any sense.
Either use regular scala class either keep these fields public.